operations/helm/charts/grafana-agent/values.yaml

```diff
@@ -99,6 +99,8 @@ image:
   # -- (string) Grafana Agent image tag. When empty, the Chart's appVersion is
   # used.
   tag: null
+  # -- Grafana Agent image's digest. When set, will override `image.tag`.
+  digest: null
```

Comment:
If you check one of the Agent images on [Dockerhub](https://hub.docker.com/layers/grafana/agent/main/images/sha256-b60321251257feddb85df35faac9e09dd07de2b628bfcf2c6e6449ee5ae56411?context=explore), the digest is reported on the form of `sha256:b6032...` which wouldn't work if copy/pasted here as it would end up as `@sha256:sha256:b6032...`

I think we should either a) document the format we expect, or b) use the `substr` function to detect whether the `sha256:` prefix already exists and only add it then. WDYT?

Reply:
Im thinking of a mixture of both. We clarify acceptable formats for commit I'd: either "sha256:XYZ" or "XYZ", and also perform input modification with `substr` for first format

Reply:
hi there, I've pushed new commits to illustrate. PTAL when you have time. Thanks!